realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -167,8 +167,21 @@ public void removeChangeListener(RealmChangeListener listener) {
     }
 
     /**
-     * Returns an Rx Observable that monitors changes to this Realm. It will emit the current state when subscribed
-     * to.
+     * Returns an RxJava Observable that monitors changes to this Realm. It will emit the current state
+     * when subscribed to. Items will continually be emitted as the Realm is updated -
+     * {@code onComplete} will never be called.
+     *
+     * If you would like the {@code asObservable()} to stop emitting items you can instruct RxJava to
+     * only emit only the first item by using the {@code first()} operator:
+     *
+     * <pre>
+     * {@code
+     * realm.where(Foo.class).findAllAsync().asObservable()
```

Comment:
I would adjust the example to fit the context. In this one, we are observing on a Realm:

```
realm.asObservable().first()
```